roles/kubernetes/node/tasks/facts.yml

```diff
@@ -17,8 +17,13 @@
 - name: Gather cgroups facts for crio
   when: container_manager == 'crio'
   block:
+  - name: Set crio_status_command for crio
+    set_fact:
+      crio_status_command: >-
+        {%- if crio_version is version("v1.29.0", operator=">=") -%}crio status{%- else -%}crio-status{%- endif -%}
```

Comment:
HI @tmurakam 

Thanks a lot for the PR. ❤

Does the `crio_status_command` duplicate with the var in `roles/container-engine/cri-o/vars/v1.29.yml`?
Should we remove one of them :-)

Reply:
Yes, I want to remove one of them. But they are defined in different roles.
How can I do this? Shall I move the declaration to kubespray-defaults role?

Reply:
The variable can be used in kuberentes/node role, so just removed the duplication from kubernetes/node role.